uploader.json

```diff
@@ -2087,51 +2087,6 @@
             ],
             "symbolic_name": "nfs_exports_d"
         },
-        {
-            "file": "/etc/nginx/()*.conf",
-            "pattern": [],
-            "symbolic_name": "nginx_conf"
-        },
-        {
-            "file": "/etc/nginx/conf.d/()*",
-            "pattern": [],
-            "symbolic_name": "nginx_conf"
-        },
-        {
-            "file": "/etc/nginx/default.d/()*",
-            "pattern": [],
-            "symbolic_name": "nginx_conf"
-        },
-        {
-            "file": "/etc/opt/rh/rh-nginx()*/nginx/()*.conf",
```

Comment:
@huali027 please leave all of these in `uploader.json` as glob will not work in this version of the json.  Please change this to `"file": "/etc/opt/rh/rh-nginx()*/nginx/*.conf"`

Reply:
@bfahr, I tested "/etc/opt/rh/rh-nginx()*/nginx/*.conf", it will raise the following error
```
>>> re.match('*.conf', 'a.conf')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/Cellar/python@2/2.7.15_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/re.py", line 141, in match
    return _compile(pattern, flags).match(string)
  File "/usr/local/Cellar/python@2/2.7.15_3/Frameworks/Python.framework/Versions/2.7/lib/python2.7/re.py", line 251, in _compile
    raise error, v # invalid expression
sre_constants.error: nothing to repeat
```
and I tested it with insights-client, it is the same.
so we can change it to "/etc/opt/rh/rh-nginx()*/nginx/.*.conf".
I tested it locally by changing "/etc/nginx/()*.conf" to "/etc/nginx/.*.conf", it works.
 However, os.path.isdir(dir_name), the dir_name doesn't support wildcard, so all the directory patch including "*" won't work. Maybe we can only inogre this in the older version.

Reply:
The uploader.json file is only used by the V1 client.  See the [code here](https://github.com/redhataccess/insights-client/blob/50498228de627296275d5df5a233f0de55f9e616/redhat_access_insights/utilities.py#L123) that the client uses to expand these regular expressions.  You are correct that line 2167 is not parsable by the client.  The reason for this is that the code uses only the os.path.basename as an argument to re.match, so the directory regex will never match anything.  Line 2152 should be `"file": "/etc/nginx/().*.conf"`.

Reply:
@bfahr, right. For line 2152, will we keep ()? I think it has no meaning actually, right?

Reply:
@huali027 I realize it may work in the test, but we have been told by the client team that it needs to be there, probably for older version of the V1 client.  So please add them.

Reply:
@bfahr, sure, I'll update the PR to keep it, thanks.